isis/src/control/objs/BundleAdjust/BundleAdjust.cpp

```diff
@@ -3110,7 +3114,10 @@ namespace Isis {
    *                           -Wformat-security warning during the build.
    */
   void BundleAdjust::outputBundleStatus(QString status) {
-    if (QCoreApplication::applicationName() != "ipce") {
+    if (iApp != NULL && QCoreApplication::applicationName() != "ipce") {
+      printf("%s", status.toStdString().c_str());
+    }
+    else (iApp == NULL) { // in function call
       printf("%s", status.toStdString().c_str());
```

Comment:
It looks like this does the same thing in the if case and the else case.

Reply:
kindof? the first iApp != null is mostly short circuiting the QCoreApplication::applicationName() as it causes a segfault when calling jigsaw as a function. I think I should just swap these cases and remove the superfluous  iApp != Null check